src/docker-compose/docker-compose-wavefront.yml

```diff
@@ -14,6 +14,9 @@ services:
       - MANAGEMENT_WAVEFRONT_METRICS_EXPORT_APITOKEN=${WAVEFRONT_KEY:?WAVEFRONT_KEY is not set!}
       - MANAGEMENT_WAVEFRONT_METRICS_EXPORT_URI=${WAVEFRONT_URI:-https://vmware.wavefront.com}
       - MANAGEMENT_WAVEFRONT_METRICS_EXPORT_SOURCE=${WAVEFRONT_SOURCE:-scdf-docker-compose}
+      - SPRING_CLOUD_DATAFLOW_APPLICATIONPROPERTIES_STREAM_MANAGEMENT_TRACING_ENABLED=true
+      - SPRING_CLOUD_DATAFLOW_APPLICATIONPROPERTIES_STREAM_MANAGEMENT_TRACING_SAMPLING_PROBABILITY=1.0
```

Comment:
The reasoning to enable tracing for all the requests (1.0) here is to help the users get a better understanding on the tracing capability?

Reply:
Otherwise it is 10% sampled and they say "where are my traces".  We can adjust or they can adjust when they install.

Reply:
Created https://github.com/spring-cloud/spring-cloud-dataflow/issues/5928 to follow up